Vizzuality/widget-editor
# Fix an issue where the boundaries layer would not be displayed on the initial load of the map

This PR fixes an issue where, when the widget-editor would restore a map widget with the boundaries layer toggled on, the layer would not be displayed on the initial render.

The issue was due to a typo.

## Testing instructions

1. Open `src/adapters/rw-adapter/src/index.ts` and replace `api.resourcewatch.org` by `staging-api.globalforestwatch.org` in the two class members `endpoint` and `dataEndpoint`
2. Restore this widget: `682b7153-c6f8-483d-b803-41ce942f6493` (dataset: `a9e33aad-eece-4453-8279-31c4b4e0583f`)
3. Open the playground

When the map is rendered, the boundaries layer must be displayed on the map. It is shown as white lines separating the countries.


## Pivotal Tracker

[Task](https://www.pivotaltracker.com/story/show/175039597).
